security/security_policy_compliance/policy-checks.sh

```diff
@@ -1,25 +1,27 @@
 #!/bin/sh
 
-function start { 
+function start {
   echo "*****************************************************"
-  printf "* $1..."; 
+  printf "* $1...\n";
 }
 
 function fail {
   DETAILS=$1
   HELP_URL=$2
 
-  echo "❌";
-  echo "${DETAILS}"
-  echo ""
+  echo "❌ ${DETAILS}\n"
 }
 
 function success {
   DETAILS=$1
 
-  echo "✅";
-  echo "${DETAILS}"
-  echo ""
+  echo "✅ ${DETAILS}\n";
+}
+
+function get_volume_attribute {
+  NEEDLE=$1
+  HAYSTACK=$2
+  echo $(echo "$HAYSTACK" | grep -e "$NEEDLE:" | sed "s/$NEEDLE: *//" | sed 's/^ *//g')
```

Comment:
FWIW, a fancy way to split the line is with `cut`
```suggestion
  echo $(echo "$HAYSTACK" | grep -e "$NEEDLE:" | cut -d : -f2 | sed 's/^ *//g')
```

Reply:
Awesome - thanks Gary! All updated in 8f4ec438d0c0bbf3d838f1dfe1370060313eca88.